### PR TITLE
Support Release: build debian packages parallelly 

### PR DESCRIPTION
**Pull request by PengTian0**
**Background**
One of the step of release is building debian packages.
This PR is used to implement the step.

**Implementation**
1. check out repositories according to a mainfest file.
2. run ```HWIMO-BUILD``` script under on-xxx repositories to build on-xxx.deb packages in parallel.
3. update debian/control of RackHD with the version of on-xxx.deb.
    In this way, RackHD.deb depends on the specify version of on-xxx.deb
4. run ```HWIMO-BUILD``` script under RackHD to build RackHD.deb

**Test**
The test jenkins job:
http://rackhdci.lss.emc.com/job/Maglev-BRI-Test/job/release/job/MBU1/

### CODE REVIEW

**Comment by PengTian0** [comment]
@panpan0000 @changev @anhou @yyscamper

File: manifest-build-tools/application/make_debian_packages.py [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+
+    parser.add_argument('--force',
+                        help="Overwrite a directory even if it exists",
+                        action="store_true")
+
+    parsed_args = parser.parse_args(args)
+    parsed_args.is_official_release = common.str2bool(parsed_args.is_official_release)
+    return parsed_args
+
+def update_rackhd_control(top_level_dir, is_official_release):
```
Comment (panpan0000) [inline manifest-build-tools/application/make_debian_packages.py]: please add verbose comment for this function.
what it's used for ..
how it's implemented..
from where  RackHD gets sub-on-xxx's deb package version

File: manifest-build-tools/application/make_debian_packages.py [hunk 2]
```diff
@@ -0,0 +1,232 @@
...
+    Build debian packages
+    """
+    try:
+        repos = get_build_repos(build_directory)
+        for repo in repos:
+            repo_dir = os.path.join(build_directory, repo)
+            generate_version_file(repo_dir, is_official_release)
+
+        # Build Debian packages of repositories except RackHD
+        repos.remove("RackHD")
```
Comment (panpan0000) [inline manifest-build-tools/application/make_debian_packages.py]: I think most different part of your PR than original OnRack code are here , right ?
original OnRack mature code should be good.
for this new part here, If it's workable and I'm fine with it.

**Comment by panpan0000** [comment]
:+1:

But I suggest your comments are more meaningful in the future ( assume the reader doesn't understand the context.  and for example : ```:param is_official_release: If true, this release is official release``` can be better to tell people what if this flag is true, what will happen ?)

**Comment by changev** [comment]
Most of the code has been proved valid in onrack ci system.
The changes also LGTM
:+1:

**Pull request merged**